modules/discoveryBidAdapter.js

```diff
@@ -547,6 +557,33 @@ export const spec = {
     return bidResponses;
   },
 
+  getUserSyncs: function (syncOptions, serverResponse, gdprConsent, uspConsent, gppConsent) {
+    setPmgUid(pmgUid); // set cookie
```

Comment:
please update to check if cookies are enabled in `syncOptions` before setting, should move to the conditional on line 577.

Reply:
Sorry, the setPmgUid function is for setting the ID into the first party cookie, not for syncing the ID with a third party. I don’t understand the relationship between setcookie and syncOptions. Could you please provide me with more details?🙏

Reply:
@ncolletti Hello, it’s been a month since this PR was submitted. Could you please review it as soon as possible?Thank you very much. 🙏

Reply:
@ecoeco163 Apologies for the back and forth, I reviewed your implementation of the set cookie and agree it's OK to leave it where it was prior. Please revert the most recent commit and I will approve the PR 👍

Reply:
@ncolletti Hi, I have change it.please review again.Thanks.😊